content/2016/12/30/initial-letter.rst

```diff
@@ -0,0 +1,138 @@
+public: yes
+author: stacy
+tags: [css, design]
+image:
+  - src: '2016/drop-caps.jpg'
+summary: |
+  Using an emphasized initial letter is a technique that has been used for
+  centuries. Let's dive in to see some ways you can apply it to your project
+  to help guide your reader with visual heiarchy.
+
+
+What's Old is New: Drop Caps in CSS
+===================================
+
+We have some pretty great new(ish) properties we can use in CSS to make our
+designs more unique and help provide useful visual cues for your audience. One
+of these newer techniques can be used to create the drop caps that we know and
+love often found in print design. This technique has been used for centuries to
+bring emphasis to a new section of text and we are finally getting a decent way
+to easily apply this with CSS.
+
```

Comment:
In the paragraph above you use "we" and "our" in all, but one place. I'd change the "your" to "our" for consistency.